packages/web/src/components/Contributions/ContributionsBox.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Box, Link } from '@chakra-ui/react';
+
+export interface ContributionsBoxProps {
+  cbox: {
```

Comment:
This isn't really readable (I have no idea what a `cbox`, but it also doesn't add value. Can you remove that and then destructure all the things you need on line 16?

Reply:
the purpose of `cbox` is for passing element from the database in `contributions.tsx` and write more readable and easier test case

Reply:
I renamed it with a more meaningful name `contributionBox`